src/common/filesystem.c

```diff
@@ -1614,6 +1616,23 @@ FS_GetNextRawPath(const char* lastRawPath)
 	return NULL;
 }
 
+#ifdef _MSC_VER // looks like MSVC/the Windows CRT doesn't have basename()
+// returns the last part of the given pathname, after last (back)slash
+// NOTE: this is not a fully compliant basename() implementation, as it doesn't
+//       handle trailing (back)slashes at all - because we don't need that.
+static char* basename( char* n )
```

Comment:
What about turning this function into a global Q_basename() function?

Reply:
Also a possibility - but I'm not sure how much sense it makes as long as basename() is only used in this one place and a custom implementation is only needed for one platform/compiler